crates/re_viewport/src/space_view.rs

```diff
@@ -106,17 +110,47 @@ impl SpaceViewBlueprint {
         let mut contents = SpaceViewContents::new(id);
         contents.insert_entities_according_to_hierarchy(queries_entities, space_path);
 
+        // TODO(jleibs): Need to create a real query here
+        let queries = vec![DataQueryId::invalid()];
+
         Self {
             display_name,
             class_name: space_view_class,
             id,
             space_origin: space_path.clone(),
             contents,
+            queries,
             entities_determined_by_user: false,
             auto_properties: Default::default(),
         }
     }
 
+    pub fn try_from_db(path: &EntityPath, blueprint_db: &StoreDb) -> Option<Self> {
+        let SpaceViewComponent {
+            display_name,
+            class_name,
+            space_origin,
+            entities_determined_by_user,
+            contents,
+        } = blueprint_db
+            .store()
+            .query_timeless_component::<SpaceViewComponent>(path)
+            .map(|c| c.value)?;
```

Comment:
won't that silently swallow deserialization errors too?

Reply:
There's a warning under the hood.